addons/website/views/snippets/snippets.xml

```diff
@@ -410,6 +410,37 @@
          data-selector='.accordion > .card'
          data-drop-in='.accordion:has(> .card)'/>
 
+    <div data-js="Accordion"
+         data-selector='.s_faq_collapse'>
```

Comment:
No need to go over two lines when not needed (same comment for the rest of the file)